README.md

```diff
@@ -133,8 +133,8 @@ You can create an empty `tagfile` with no extention in any folder you want to ap
 
 | Flag/File | Description | Notes |
 |------|-------------|-------|
-| `WABBAJACK_INCLUDE` | All files in this folder will be inlined into the `.wabbajack` file | |
-| `WABBAJACK_NOMATCH_INCLUDE` | All files in this folder will be inlined into the `.wabbajack` file even if Wabbajack did not found a match for them. | Useful for generated files.|
+| `WABBAJACK_INCLUDE` | All files in this folder will be inlined into the `.wabbajack` file | Can lead to large `.wabbajack` files if used unsparingly |
```

Comment:
```suggestion
| `WABBAJACK_INCLUDE` | All files in this folder will be inlined into the `.wabbajack` file | **DO NOT USE ON DOWNLOADED MODS** As it would result in you distributing that mod. Can lead to large `.wabbajack` files if used unsparingly |
```